src/lib/form-field/form-field.ts

```diff
@@ -419,6 +419,9 @@ export class MatFormField extends _MatFormFieldMixinBase
         // getBoundingClientRect isn't available on the server.
         return;
       }
+      if (!document.contains(this._elementRef.nativeElement)) {
```

Comment:
This one'll probably throw in Universal.

Reply:
The line above is returning if we're not in the browser